payments/models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class Subscription(models.Model):
+    owner = models.OneToOneField(User)
```

Comment:
using `primary_key=True` here helps avoids issues if/when the auth_user_seq_id and payments_subscription_seq_id sequences get out of sync - it is quite confusing when te PK is (for example) 200 but the user_id is 201..